libcaf_test/caf/test/fixture/deterministic.cpp

```diff
@@ -15,12 +17,90 @@
 #include "caf/scheduled_actor.hpp"
 
 #include <numeric>
+#include <string>
 #include <string_view>
+#include <utility>
 
 using namespace std::literals;
-
 namespace caf::test::fixture {
 
+namespace {
+
+using namespace caf;
+
```

Comment:
```suggestion
```

Leftover? We're already in namespace `caf`.

Reply:
Yes, I think it got added back again while resolving the merge conflicts. I am removing it now.